src/App.tsx

```diff
@@ -1,14 +1,55 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { TodoList } from './components/todo/TodoList';
+import { Todo } from './types/Todo';
+import { Footer } from './components/footer/Footer';
+// eslint-disable-next-line max-len
+import { ErrorNotification } from './components/errorNotification/ErrorNotification';
+// eslint-disable-next-line max-len
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMesage, setErrorMesage] = useState<string>('');
+  const [statusFilter, setStatusFilter] = useState<string>('all');
```

Comment:
```suggestion
  const [errorMesage, setErrorMesage] = useState('');
  const [statusFilter, setStatusFilter] = useState('all');
```
it's redundant to specify types for primitives